pyfstat/core.py

```diff
@@ -150,7 +150,8 @@ def read_par(
         )
         return params_dict
 
-    def translate_keys_to_lal(self, dictionary):
+    @staticmethod
+    def translate_keys_to_lal(dictionary):
```

Comment:
@dbkeitel This may be controversial.
I just think about those times in which `lal` functions have to be invoked via `subprocess` when `PyFstat` is already on use: Then it makes sense to have this method available (otherwise you'd have to code it yourself *every single time*).

Reply:
Just so I remember what this does, after chat clarification: The idea is that you can do
```
from pyfstat import BaseSearchClass
translated_dict = BaseSearchClass.translate_keys_to_lal(mydict)
```
without having init-ed an object.

The alternative would be to make it a helper_function, but since as you say it won't be a commonly needed feature, this should suffice.